gradle.properties

```diff
@@ -0,0 +1,2 @@
+group=org.springldap
+description=Spring Framework Docs Site
```

Comment:
I think this should be Spring LDAP. I also wonder if `group` is correct, but I don't think I understand it's meaning yet.